documentation/typespec-rest-api-dev-process.md

```diff
@@ -45,20 +33,39 @@ Once VSCode reopened in Container, you can run any of the program below in the V
 - Ensure you can run TypeSpec command within the repo folders.
 
 ```
-   tsp --version 
+   npx tsp --version 
```

Comment:
Do you want folks to use npx here as opposed to having tsp installed on the path?

Reply:
I think we have been saying to discourage folks installing TSP globally and only rely on the `npm install` in the repo. Without `npx`, locally installed from node_modules' tsp command won't work.

Reply:
Ok I wasn't aware of that is that in the docs that? Also should we add tsp to the path for the docker/devcontainer so that it is slightly easier or should we still require npx.